tests/arch/quicklogic/logic.ys

```diff
@@ -0,0 +1,16 @@
+read_verilog v/logic.v
```

Comment:
Please use `common/logic.v` instead of unnecessarily bundling a different copy of it.